ruby/ql/lib/change-notes/2022-03-21-regex.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The `ParseRegExp` and `RegExpTreeView` modules are now "internal" modules, users should use `codeql.ruby.Regexp` instead.
```

Comment:
I know this is pedantic, but I dislike [comma splices](https://en.wikipedia.org/wiki/Comma_splice) 😛 
```suggestion
* The `ParseRegExp` and `RegExpTreeView` modules are now "internal" modules. Users should use `codeql.ruby.Regexp` instead.
```